examples/tracing/cache_miss.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# cache_miss.py Example of using BPF perf event.
+#               For Linux, uses BCC, eBPF. Embedded C.
+#
+# REQUIRES: Linux 4.7+ (BPF_PROG_TYPE_PERF_EVENT support).
+#
+# Copyright (c) 2016 Facebook, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 18-Oct-2016    Teng Qin    Created this.
+
+from __future__ import print_function
+import argparse
+from bcc import BPF, BPFPerfType, BPFPerfHWConfig
+import signal
+from time import sleep
+
+parser = argparse.ArgumentParser(
+    description="Summarize cache misses by PID or stack trace",
+    formatter_class=argparse.RawDescriptionHelpFormatter)
+parser.add_argument(
+    "--duration", default=10, type=int, help="Time to run, in seconds")
+parser.add_argument(
+    "--sample_period", default=0, type=int,
```

Comment:
I think this is better called "--sample_count", with a shortened version as well, "-c". This also makes it similar to "perf record".